README.md

```diff
@@ -1,15 +1,19 @@
 <div align="center">
 
-# `deltamin`
+# `deltamin` Δ
 
 Minify your Delta format data
 before sending it down the wire
 or saving it to a datastore.
 
-[![Build Status](https://img.shields.io/travis/com/dwyl/deltamin/master.svg?style=flat-square)](https://travis-ci.com/dwyl/deltamin)
-[![codecov.io](https://img.shields.io/codecov/c/github/dwyl/deltamin/master.svg?style=flat-square)](http://codecov.io/github/dwyl/deltamin?branch=master)
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/dwyl/terminate/js.yml?label=build&style=flat-square&branch=main)
```

Comment:
Mayhaps you've meant `![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/dwyl/deltamin/js.yml?label=build&style=flat-square&branch=main)`?

This label is using `dwyl/terminate`'s repo handle.

Reply:
Good catch. Copy-paste fail. Fixing.

Reply:
I was just testing your attention to detail. 😜 
Fixed.